OpenRA.Game/FileFormats/Png.cs

```diff
@@ -25,6 +25,12 @@ namespace OpenRA.FileFormats
 {
 	public class Png
 	{
+		public enum Compression : byte
+		{
+			BEST_COMPRESSION = Deflater.BEST_SPEED,
```

Comment:
These are backwards?!

Reply:
oh, that's hillarious. I didn't test with HD sheets so didn't notice it was wrong